clients/cobol-lsp-vscode-extension/src/__tests__/LanguageClientServiceTest.spec.ts

```diff
@@ -51,29 +51,29 @@ const SERVER_ID = "COBOL";
 
 beforeEach(() => {
     jest.clearAllMocks();
-});
-
-describe("LanguageClientService negative scenario.", () => {
-
-    test("LSP port not defined and jar path doesn't exists", async () => {
-        await expect(new LanguageClientService(undefined).checkPrerequisites())
-        .rejects.toThrowError("LSP server for COBOL not found");
-}, 10000);
+    vscode.workspace.getConfiguration(expect.any(String)).get = jest.fn().mockReturnValue(0);
 });
 
 const SERVER_STARTED_MSG = "server started";
 const SERVER_STOPPED_MSG = "server stopped";
 describe("LanguageClientService positive scenario", () => {
-
     beforeEach(() => {
-        jest.clearAllMocks();
         languageClientService = new LanguageClientService(middleware);
         new JavaCheck().isJavaInstalled = jest.fn().mockResolvedValue(true);
+        vscode.workspace.getConfiguration(expect.any(String)).get = jest.fn().mockReturnValue(0);
         fs.existsSync = jest.fn().mockReturnValue(true);
     });
 
     test("Test LanguageClientService checkPrerequisites passes", async () => {
-        expect(async () => await languageClientService.checkPrerequisites()).not.toThrowError();
+        let message = false;
+        fs.existsSync = jest.fn().mockReturnValue(true);
+        vscode.workspace.getConfiguration(expect.any(String)).get = jest.fn().mockReturnValue(9999);
+        try {
+            await languageClientService.checkPrerequisites();
+        } catch (error) {
+            message = error;
+        }
+        expect(message).toBeFalsy();
```

Comment:
Why not to keep `.not.toThrowError()`?

Reply:
yes, that was the previous implementation. 
`await expect(languageClientService.checkPrerequisites()).not.toThrowError();` should work.
But was failing on my machine.  So I took a long route to resolve everything at code and use jest to assert.
No clue why. 🤕